MtfhReportingDataListener/Infrastructure/KafkaInitialisation.cs

```diff
@@ -0,0 +1,50 @@
+using Confluent.Kafka;
+using Microsoft.Extensions.DependencyInjection;
+using MtfhReportingDataListener.Boundary;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+
+namespace MtfhReportingDataListener.Infrastructure
```

Comment:
Can any of this go into a shared package in the future?

Reply:
Possibly, let's see how this spike goes